src/pages/documentation/migrate-to-android.md

```diff
@@ -16,18 +16,21 @@ At this time, the previous versions of the Android Mobile SDKs may be switched o
+| campaign        | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaign.svg?logo=android&logoColor=white&label=campaign)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaign)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| campaignclassic | [![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/campaignclassic.svg?logo=android&logoColor=white&label=campaignclassic)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/campaignclassic)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
 
 ### Manual library import
```

Comment:
the changes look good to me. aside from that, should we also include all the extensions to the gradle example below?